types/integra/aquastream/device.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/device.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Device",
+  "type": "object",
+  "properties": {
+    "manufacturerCode": {
+      "title": "Manufacturer Code",
+      "type": "string",
+      "description": "Manufacturer Code",
+      "hideFromKpis": true
+    },
+    "moduleNumber": {
+      "title": "Module number",
+      "type": "string",
+      "description": "Module number",
+      "hideFromKpis": true
+    },
+    "versionNumber": {
+      "title": "Version number",
+      "type": "integer",
+      "description": "Version number",
+      "hideFromKpis": true
+    },
+    "systemComponent": {
+      "title": "System component",
+      "type": "integer",
+      "description": "System component",
+      "hideFromKpis": true
+    },
+    "meterAddress": {
+      "title": "Meter address",
+      "type": "string",
+      "description": "Meter address",
+      "hideFromKpis": true
+    },
+    "meterVersion": {
+      "title": "Meter version",
+      "type": "integer",
+      "description": "Meter version",
+      "hideFromKpis": true
+    },
+    "waterType": {
+      "title": "Water type",
+      "type": "integer",
+      "description": "Either warm or cold",
```

Comment:
use an ENUM? or describe in which case it is 0 and 1

Reply:
Theres sadly no description to this datapoint.